infra/packaging/preset/20200630

```diff
@@ -28,11 +28,14 @@ function preset_configure()
   REQUIRED_UNITS+=("record-minmax" "circle-quantizer")
   REQUIRED_UNITS+=("one-cmds")
 
+  NPROC=$(cat /proc/cpuinfo | grep -c processor)
```

Comment:
Could you please allow users to override this value? This value in Circle CI is 80, and.... it will cause out of memory :(